revolv/templates/base/partials/foot.html

```diff
@@ -51,15 +51,15 @@
           people and communities to invest collectively in renewable energy.
           All donations are tax-deductible.
         </p>
-		<dl>
-			<dd><a href="https://old.re-volv.org/">Visit our old website</a></dd>
-		</dl>
-		<dl>
-			<dt><a href="http://goo.gl/forms/S1TEKTNgNz">How do you like our new website?</a></dt>
-		</dl>
-		<p>
-			Take a short survey and let us know. We would love to hear your feedback!
-		</p>
+		    <dl>
```

Comment:
These changes are not necessary, and the preferred indentation level is four spaces. Please revert these changes. You don't need to change everything to use four-space indentation if it doesn't, but please do not change indentation if you don't need to.

Part of the reason this is hard to read is because it's indented with tabs (unfortunate, I know). However, let's not change it if we don't have to.